src/list-data/request-action-mapping.ts

```diff
@@ -100,22 +100,22 @@ export const BusinessLookupRequestActions = [
 /** Entity types that require business lookup. */
 export const BusinessLookupEntityTypes = [
   EntityTypes.A,
-  EntityTypes.BC,
-  EntityTypes.BEN,
+  EntityTypes.BC, // Benefit Company
+  EntityTypes.BEN, // invalid?
```

Comment:
I think this is invalid but I'm not about to remove it in a hotfix without fully understanding how this list is used 😛 

I also checked the other entries in this list, and I think they're OK, but there may be some types missing.

Reply:
Yeah I agree I had a brief look through namex and I don't believe BEN is valid, but it is probably not worth removing it on a hotfix.